OpenUpSA/wazimap-ng-ui
# open graph metadata webflow changes

## Description
imported webflow export containing favicon and page title changes. added a gui test step to check the default title

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-292

## How is it tested automatically?
* added a test step into `__tests__/gui/error.feature`

## How should a reviewer test it locally
* open the deploy preview
* _before_ the profile is loaded, confirm that the title is `Wazimap NG` and not `Wazimap NG 2021`

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/167954674-5916329c-8c40-40ef-b637-b576a4cd5011.png)


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
